tests/ci/report.py

```diff
@@ -379,15 +458,15 @@ def create_test_html_report(
 
 
 def create_build_html_report(
-    header,
-    build_results,
-    build_logs_urls,
-    artifact_urls_list,
-    task_url,
-    branch_url,
-    branch_name,
-    commit_url,
-):
+    header: str,
+    build_results: BuildResults,
+    build_logs_urls: List[str],
+    artifact_urls_list: List[List[str]],
+    task_url: str,
```

Comment:
This one if GitHub actions link?

Reply:
Here where it is used https://github.com/ClickHouse/ClickHouse/pull/44883/files/e1dc38f0fe2d41c54ea78ef7ff164778a30948b0#diff-e531990fe28f8c07670f4c4963ecbe0381b145ec84b586ad0f95e113b9f074efR228